### ISSUE TITLE
React 18 support

### ISSUE THREAD

**Issue by Semigradsky** [recharts/react-smooth#57]

**Comment by Semigradsky** [comment]
@arcthur could you please update it?

**Comment by avshyz-gs** [comment]
Hi, any update on this? It really screws us over since we can't update to react 18 without it :\

**Comment by brookjordan** [comment]
Not ideal, but anyone hitting these issues may be able to resolve the react types for now. That is, in your package.json, add:
```json
  "resolutions": {
    "node-fetch": "^2.6.7",
    "postcss": "^8.3.11",
    "@types/react": "^18.0.21"
  },
```

**Comment by ckifer** [comment]
2.0.1 was released on June 27, 2022 which added support for react 18. Is still an issue?

Next major version we will drop support for React < 16.8 here and in Recharts so we can upgrade `react-transition-group`

**Comment by Semigradsky** [comment]
Cool. Closed in favor of #33
